core/profiles/standard/config/layout.layout.home.json

```
@@ -114,19 +115,27 @@
                 }
             }
         },
-        "578243cf-a4c8-46ad-9504-04aaf4ec9ccc": {
-            "plugin": "views:promoted-block",
+        "209098f9-8e8f-4be6-8000-0033df223bdd": {
+            "plugin": "node:content",
             "data": {
-                "module": "views",
-                "delta": "promoted-block",
+                "module": "node",
+                "delta": "content",
                 "settings": {
-                    "title_display": "default",
+                    "title_display": "none",
                     "title": "",
                     "style": "default",
                     "block_settings": [],
-                    "contexts": []
+                    "contexts": [],
+                    "admin_label": "Home Page Content",
+                    "admin_description": "A node with content for the home page.",
```

We should not be using the word "node" in user-facing text. How about this instead?:

```suggestion
                    "admin_description": "A piece of content used as the home page.",
```